sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -355,9 +356,9 @@ export interface Model {
 
 // @public
 export interface ModelInfo {
```

Comment:
and this one is called `CustomFormModelInfo`.

Reply:
There's already another `CustomFormModelInfo`. I should be able to remove this one in PR #8933